src/app/models/tag.model.ts

```diff
@@ -5,4 +5,5 @@ export interface TagModel extends Item {
   id: string;
   title: string;
   type: ItemType.tag;
+  checked?: boolean
```

Comment:
you can use ES6.symbol for internal Frontend.keys to avoid collisions with possible keys from the Backend

like:
const keyChecked = Symbol("checked");
...
[keyChecked]?: boolean

but i do not know how it works in Angular